Core/src/main/java/dk/sdu/mmmi/modulemon/managers/GameViewManager.java

```diff
@@ -35,8 +35,8 @@ public void setState(IGameViewService state, boolean disposeCurrent) {
         currentGameState.init(this);
```

Comment:
Please also change this variable name from `currentGameState` to `currentGameView`